### ISSUE TITLE
Remmina preferences are not being saved 

### ISSUE THREAD

**Issue by Neldogz** [FreeRDP/Remmina#627]
This is the same issue as documented in: https://github.com/FreeRDP/Remmina/issues/391 which as since been closed. 

Changes made within the Remmina preferences dialog are not being saved. This applies to both global and connection specific preferences.  

Linux computer1 4.1.4-1-ARCH #1 SMP PREEMPT Mon Aug 3 21:30:37 UTC 2015 x86_64 GNU/Linux
GNOME Shell 3.16.3 
Remmina version 1.1.2 - Installed from the official Arch repositories. 

Attempting to RDP to a Windows 7 64-bit host.

**Comment by Neldogz** [comment]
Followed the directions as recommended per the following article https://github.com/FreeRDP/Remmina/wiki/Compile-Remmina-on-Arch-Linux but still unable to save preferences.

**Comment by giox069** [comment]
1. Check that you are using remmina version 1.2.X: check on the menu help->about
2. Check that you have write permissions on the ~/.remmina directory: try to create a file there and then delete it. Also check for ~/.config/remmina directory
3. Try to launch remmina from a terminal window, and see if you get some message when saving preferences or profiles.
4. If none works, backup and then delete the directories ~/.remmina and ~/.config/remmina. Kill remmina with "killall remmina" and restart it.

**Comment by Neldogz** [comment]
Thank you for your response but these steps did not resolve the issue. 

1. I tested these steps using Remmina version 1.2.0-rcgit.3. 
2. Confirmed write permissions within both the ~/.remmina and ~/.config/remmina directories
3. Launched remmina from a terminal window and tried to make changes within the RDP tab but they did not take. Only messages displayed in the terminal are as follows:
(remmina:6870): GLib-GObject-WARNING **: The property GtkAlignment:right-padding is deprecated and shouldn't be used anymore. It will be removed in a future version.
4. Deleted the directories, killed the remmina processes and then started remmina again. Same problem unfortunately.

**Comment by giox069** [comment]
Which preferences are not saved ? All preferences ? Or only a subset of RDP preferences as in #391 ?
For example, from the main window choose Edit->Preferences, on the "Options" tab change "Scale quality" to "Tiles", change it. Is it saved ? Also try to see what's inside `~/.remmina/remmina.pref`, look at the line `scale_quality=` and see if it changes when you change the scale quality from the remmina preference windows.

**Comment by Neldogz** [comment]
Within the RDP tab for global preferences dialog I failed to see that the quality settings drop down menu now states: "Choose a quality level to edit". I now realize that I cannot set one of the quality setting profiles as a global default. Once edited the quality setting profile can then be applied to individual hosts. The menu selections are indeed working properly in version 1.2.0-rcgit.3. I appologize for my misinterpretation.

I went back to version 1.1.2 and confirmed that the issue originally described in https://github.com/FreeRDP/Remmina/issues/391 is present and have raised a ticket with the Arch maintainer for this package. Hopefully we can get version 1.2.0 packaged for Arch. Thanks again for your help!
